src/libstd/net/mod.rs

```diff
@@ -111,3 +111,13 @@ impl Iterator for LookupHost {
 pub fn lookup_host(host: &str) -> io::Result<LookupHost> {
     net_imp::lookup_host(host).map(LookupHost)
 }
+
+/// Resolve the given address to a hostname.
+///
+/// This function may perform a DNS query to resolve `addr` and may also inspect
+/// system configuration to resolve the specified address. If the address
+/// cannot be resolved, it is returned in string format.
+#[unstable(feature = "ip_addr", reason = "recent addition")]
```

Comment:
I think this is separable enough from the `ip_addr` feature that this may want to be called `lookup_addr` instead.